packages/manager/src/features/Images/ImagesLanding.tsx

```diff
@@ -537,12 +546,56 @@ export const ImagesLanding: React.FC<CombinedProps> = (props) => {
             deleted. They will be deleted after the indicated expiration date.
           </Typography>
         </div>
-        <EntityTable
-          entity="image"
-          row={autoImageRow}
-          headers={automaticHeaders}
-          emptyMessage={'No Recovery Images to display.'}
-          initialOrder={initialOrder}
+        <Table>
+          <TableHead>
+            <TableRow>
+              <TableSortCell
+                active={automaticImagesOrderBy === 'label'}
+                direction={automaticImagesOrder}
+                label="label"
+                handleClick={handleAutomaticImagesOrderChange}
+              >
+                Image
+              </TableSortCell>
+              <Hidden smDown>
+                <TableCell>Status</TableCell>
+              </Hidden>
+              <Hidden smDown>
+                <TableCell>Created</TableCell>
```

Comment:
Let's let people sort by `created` because the API supports it

```suggestion
                <TableSortCell
                  active={automaticImagesOrderBy === 'created'}
                  direction={automaticImagesOrder}
                  label="created"
                  handleClick={handleAutomaticImagesOrderChange}
                >
                  Created
                </TableSortCell>
```